standard/table.lua

```diff
@@ -219,6 +219,75 @@ function Table.map(xTable, f)
 	return yTable
 end
 
+--[[
+Extracts prefixed keys interleaved with numeric indexes from an arguments
+table, and applies a transform to each key or index.
+
+Used for template calls that support both prefixed and indexed params. See
+Module:ParticipantTable/Starcraft, Module:GroupTableLeage for examples of how
+it is used.
+
+Example:
+In the template call
+{{Foo
+	|A
+	|p2=B
+	|C
+	|player4=D
+}}
+
+Table.mapInterleavedPrefix(args, {'p', 'player'}, f)
+will invoke
+
+f(1, 1)
+f('p2', 2, 'p')
+f(3, 3)
+f('player4', 4, 'player')
```

Comment:
These resuls don't make sense to me, should they not be `A`, etc?

Reply:
I think `mapArgumentsByPrefix` makes more sense as name

Reply:
No, I think the word "interleaved" should not be used, it means nothing to me when I read the function name

Reply:
Yea I get the idea, but simple is better